dime-research-standards/pillar-1-research-ethics/research-ethics-resources/informed-consent-templates/informed-consent-template-written-covid.md

```diff
@@ -0,0 +1,197 @@
+# WRITTEN CONSENT
+
+**[Insert title of the study]**
+
+You have been asked to participate in a research study conducted by **[insert names and degrees of all investigators]**,
+from the **[insert institutional affiliation]**.
+You were selected as a possible participant in this study because
+**[explain succinctly and simply why the prospective subject is eligible to participate]**.
+
+The information below provides a summary of the research.
+Your participation in this research is voluntary and you can withdraw at any time.
+
+### Purpose
+
+**[Short sentence describing purpose of research. Use lay language. 1-2 sentences.]**
+
+### Study Procedures
+
+**[Brief explanation of study procedures. 1-2 sentences.]**
+
+### Risks & Potential Discomfort
+
+**[Describe any reasonable foreseeable risks, discomforts, inconveniences, and how these will be managed]**
+
+### Alternatives to Participation (remove this section if no alternatives exist)
+
+**[Describe any appropriate alternative therapeutic, diagnostic, or preventive procedures that should be considered before the
+subject decide whether or not to participate in the study.]**
+
+You should read the information below, and ask questions about anything you do not understand before deciding whether or
+not to participate.
+
+## Participation and withdrawal
+
+Your participation in this study is completely voluntary and you are free to choose whether to be in it or not.
+If you choose to be in this study, you may subsequently withdraw from it at any time without penalty or consequences of any kind.
+The investigator may withdraw you from this research if circumstances arise.
```

Comment:
does this ever happen in our context?

Reply:
I am not sure. Haven't come across a case. Removed the line for now